linode/vpcsubnet/resource_test.go

```diff
@@ -0,0 +1,206 @@
+//go:build integration
+
+package vpcsubnet_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/linode/helper"
+	"github.com/linode/terraform-provider-linode/linode/vpcsubnet/tmpl"
+)
+
+var vpcID int
+
+func init() {
+	resource.AddTestSweepers("linode_vpc_subnet", &resource.Sweeper{
+		Name: "linode_vpc_subnet",
+		F:    sweep,
+	})
+
+	client, err := acceptance.GetTestClient()
+	if err != nil {
+		log.Fatal(fmt.Errorf("Error getting client: %s", err))
+	}
```

Comment:
Oddly the client created by this helper doesn't seem to be respecting the `LINODE_URL` specified in the environment.

Reply:
Yes, I think it uses the `DefaultLinodeURL` https://github.com/linode/terraform-provider-linode/blob/8c4f240e1577d6bd97a28dc0878594facc876aea/linode/helper/config.go#L20 if the api url is not specified in the Config.

Reply:
I think we can just remove that line and everything will be fine since the `linodego` already specify this value as the default base URL.

Reply:
We probably also want to call `helper.Client()` in `InitProvider()` to unify the the behavior of the client and make `helper.Client()` as something like a SSOT.

Reply:
Let's address this in another change: https://jira.linode.com/browse/TPT-2342